src/routes/docs/tutorials/react/step-4/+page.markdoc

```diff
@@ -66,21 +67,24 @@ Now, we can use the `useUser` hook to access the user's data from any component.
 
 # Basic routing  {% #basic-routing %}
 
-First, wrap the `main` element with the `UserProvider` component.
+First, wrap the `main` element with the `UserProvider` and `IdeaProvider` component.
```

Comment:
Does the `IdeasProvider` exist at this part of the tutorial? What will happen when it's imported?